runtime/browser/runtime_url_request_context_getter.cc

```diff
@@ -139,7 +159,7 @@ net::URLRequestContext* RuntimeURLRequestContextGetter::GetURLRequestContext() {
             net::DISK_CACHE,
             net::CACHE_BACKEND_DEFAULT,
             cache_path,
-            0,
+            GetDiskCacheSize(),
```

Comment:
I don't like the idea of needlessly re-parsing command line arguments like that; isn't it easy to just do it once and store the value?